yarn.lock

```diff
@@ -3324,6 +3324,22 @@ broccoli-funnel@^3.0.0, broccoli-funnel@^3.0.2:
     path-posix "^1.0.0"
     walk-sync "^2.0.2"
 
+broccoli-funnel@^3.0.3:
+  version "3.0.3"
+  resolved "https://registry.npm.taobao.org/broccoli-funnel/download/broccoli-funnel-3.0.3.tgz#26fd42632471f67a91f4770d1987118087219937"
```

Comment:
This resolve path should be pointing at yarnpkg.com

Reply:
ok, i will update it . i use other mirror in global .